src/User/Bootstrap.php

```diff
@@ -162,6 +163,40 @@ function () use ($model) {
                 });
             }
 
+            // Initialize array of two factor authentication validators available
+            $defaultTwoFactorAuthenticationValidators = 
+               [
+                    'google-authenticator'=>[
+                        'class'=>\Da\User\Validator\TwoFactorCodeValidator::class,
+                        'description'=>Yii::t('usuario', 'Google Authenticator'),
+                        'configurationUrl'=>'user/settings/two-factor',
+                        'enabled'=>true
+                    ],
+                    'email'=>[
+                        'class'=>\Da\User\Validator\TwoFactorEmailValidator::class,
+                        'description'=>Yii::t('usuario', 'Email'),
+                        'configurationUrl'=>'user/settings/two-factor-email',
+                        // Time duration of the code in seconds
+                        'codeDurationTime'=>300,
+                        'enabled'=>true
+                    ],
+                    'sms'=>[
+                        'class'=>\Da\User\Validator\TwoFactorTextMessageValidator::class,
+                        'description'=>Yii::t('usuario', 'Text message'),
+                        'configurationUrl'=>'user/settings/two-factor-sms',
+                        // component for sending sms
+                        'smsSender'=>'smsSender',
+                        // Time duration of the code in seconds
+                        'codeDurationTime'=>300,
+                        'enabled'=>true
+                    ]
+                ];
+
+            $app->getModule('user')->twoFactorAuthenticationValidators =  
+                    ArrayHelper::merge($app->getModule('user')->twoFactorAuthenticationValidators, $defaultTwoFactorAuthenticationValidators); 
```

Comment:
This should be inverted. The settings on the module should override the default ones, as it is now it's the opposite.
```php
$app->getModule('user')->twoFactorAuthenticationValidators = ArrayHelper::merge(
     $app->getModule('user')->twoFactorAuthenticationValidators,
     $defaultTwoFactorAuthenticationValidators
); 
```